Lib/Utils/DatabaseManager.cs

```diff
@@ -148,7 +149,7 @@ public static bool Setup(string filename, DBSettings? dbSettingsIn = null)
             {
                 Connections = new List<SqlConnectionHolder>();
 
-                PopulateConnections();
```

Comment:
bool result of EstablishMainConnection is not checked

Reply:
Good catch.